PSReadLine/PSReadLineResources.Designer.cs

```diff
@@ -2067,5 +2067,26 @@ internal static string WindowSizeTooSmallForListView
                 return ResourceManager.GetString("YankPopDescription", resourceCulture);
             }
         }
+
+        /// <summary>
+        ///   Looks up a localized string similar to No help content available. Please use Update-Help.
+        /// </summary>
+        internal static string NeedsUpdateHelp {
+            get {
+                return ResourceManager.GetString("NeedsUpdateHelp", resourceCulture);
+            }
+        }
+
+        /// <summary>
+        ///   Looks up a localized string similar to Displaying help content is not supported on this console.
```

Comment:
Make sure this comment is updated.
It's unfortunately that today we still have to manually update this file. Will look into do resgen similarly to PowerShell.